snippets/base/tests/test_models.py

```diff
@@ -667,16 +667,40 @@ def test_channels(self):
         self.assertTrue(job.channels, set(['release', 'beta', 'nightly']))
 
     def test_clean(self):
-        job_clean = JobFactory.create(publish_start=datetime.utcnow() - timedelta(days=1),
-                                      publish_end=datetime.utcnow())
+        job_clean = JobFactory.create(publish_start=datetime.utcnow() + timedelta(days=1),
+                                      publish_end=datetime.utcnow() + timedelta(days=2))
         job_clean.clean()
 
-        job_dirty = JobFactory.create(publish_start=datetime.utcnow(),
-                                      publish_end=datetime.utcnow() - timedelta(days=1))
+        job_dirty = JobFactory.create(publish_start=datetime.utcnow() + timedelta(days=3),
+                                      publish_end=datetime.utcnow() + timedelta(days=2))
 
         self.assertRaisesMessage(
             ValidationError, 'Publish start must come before publish end.', job_dirty.clean)
 
+    @override_settings(SNIPPETS_PUBLICATION_OFFSET=5)
+    def test_clean_publication_offset(self):
+        utcnow = datetime.utcnow()
+
+        job_no_publish_start = JobFactory.create(publish_start=None, publish_end=None)
+        with patch('snippets.base.models.datetime') as datetime_mock:
+            datetime_mock.utcnow.return_value = utcnow
+            job_no_publish_start.clean()
+        job_no_publish_start.publish_start = utcnow + timedelta(5)
```

Comment:
These look like assignments but I think they should be assertions. Also I think you need to use the kwarg: `timedelta(minutes=5)`.

Reply:
Ermm of course. Thanks!